### DISCUSSION TITLE
Remove/Replace p tags in <li> with <span>

### DISCUSSION THREAD

**Discussion by gowrusreevathsa** [ueberdosis/tiptap#1963] [General Questions] [answered]
Hi,

List items are getting wrapped in a paragraph. I get that this is due to the `paragraph block*` content as nodes require a child node.

I am querying if I could change the **li** tag to render a SPAN instead of a P tag or remove the p tags?

HTML content from editor
```
<ul>
  <li>
    <p><strong>Name:</strong>Test</p>
  </li>
  <li>
    <p><strong>Email:</strong>test@user.com</p>
  </li>
  <li>
    <p><strong>Contact:</strong> XXXXX</p>
  </li>
</ul>

```

Replace p with span

```
<ul>
  <li>
    <span><strong>Name:</strong>Test</span>
  </li>
  <li>
    <span><strong>Email:</strong>test@user.com</span>
  </li>
  <li>
    <span><strong>Contact:</strong> XXXXX</span>
  </li>
</ul>

```

I should get rid of those p tags as I use the content in the editor to send emails. The p tags are creating additional spaces, which is not acceptable in my use case. 

Any ideas are much appreciated.

**Comment by philippkuehn** [comment]
see #118

**Comment by cadars** [comment, marked as answer]
>  The p tags are creating additional spaces

It might be easier to solve this with CSS only?
```css
li p {
  display: inline; /* so it behaves like a span */
  margin: 0;
}
```
Or, since you’re doing emails, as inline styles.
